test_runner/regress/test_s3_proxy.py

```diff
@@ -0,0 +1,56 @@
+from time import time
+
+import pytest
+from aiohttp import ClientSession
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv
+from jwcrypto import jwk, jwt
+
+
+@pytest.mark.asyncio
```

Comment:
In the storage team we generally try to avoid async IO since it's easy to stall the sole python thread and hold the GIL if any of your futures takes longer than expected. Thread pools are preferred.

Reply:
Asyncio event loop in pytest uses a thread pool

Reply:
Yeah, this wasn't a hard "don't do that", just an observation from lessons learned in the past.
Fwiw, I don't see anything wrong with the asyncio usage here, so feel free to resolve.